src/main/java/nl/b3p/tailormap/api/persistence/User.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (C) 2023 B3Partners B.V.
+ *
+ * SPDX-License-Identifier: MIT
+ */
+package nl.b3p.tailormap.api.persistence;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import java.util.HashSet;
+import java.util.Set;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import org.hibernate.annotations.Type;
+
+@Entity
+@Table(name = "users")
+public class User {
```

Comment:
Should we add a Date field to make it possible so set an expiry date for a user account? eg.
```suggestion
public class User {


  @FutureOrPresent
  private java.time.ZonedDateTime validUntil;
```

https://b3partners.atlassian.net/browse/HTM-114

Reply:
👍🏻